server/base/src/main/java/org/apache/accumulo/server/util/ManagerMetadataUtil.java

```diff
@@ -199,10 +199,10 @@ public static void replaceDatafiles(ServerContext context, KeyExtent extent,
     TabletMutator tablet = context.getAmple().mutateTablet(extent);
 
     datafilesToDelete.forEach(tablet::deleteFile);
-    scanFiles.forEach(tablet::putScan);
+    scanFiles.stream().map(StoredTabletFile::getTabletFile).forEach(tablet::putScan);
```

Comment:
Wondering if putScan needs an overloaded method, not sure if its ever updating existing metadata.

Reply:
It turns out that we only ever call this in one spot and we have a StoredTabletFile so I am just going to switch putScan() to take a StoredTabletFile instead.